compiler/README.md

```diff
@@ -62,6 +62,10 @@ options:
                         Method Name (must be defined in proto)
   -b BACKEND, --backend BACKEND
                         Backend Code Target
+  -n MOD_NAME, --mod_name MOD_NAME
```

Comment:
I assume these two options are only for gRPC backend right? Can you add an explanation here?

Reply:
The supported backends section of this readme explains them, though I could move it up.